.licensei.toml

```diff
@@ -0,0 +1,10 @@
+approved = [
```

Comment:
Should license headers be added to new files?

If so, which one?

Probably a good case for setting up the `headers` section of this file at the same time.